benchmark/framework/containerd_utils.go

```diff
@@ -258,6 +261,17 @@ func (proc *ContainerdProcess) RunContainerTaskForReadyLine(
 	if err := taskDetails.task.Start(ctx); err != nil {
 		return nil, err
 	}
+	if parseFileAccessPatterns {
+		data := bparser.FileAccessPatterns{
+			ContainerStartTime: time.Now(),
```

Comment:
Is container task started yet?

Reply:
At this point the container process should be running. It is not tracking the time of ready line. Should it?

Reply:
Are we measuring the ContainerStartTime right? This should be the time, container task starts.

Reply:
This looks like the correct place to get container start time - but I'm not sure why we want to inject the other logs here

Reply:
Is this something we need?

Reply:
The container start time is now being pulled from  containerd stderr at the time in which the `containerd.events.TaskStart` event is emitted, instead of logging the time after `task.Start` returns.